### PR TITLE
Kill the summary path

### PR DESCRIPTION
**Pull request by rozap**
* It was leading to unexpected 4x4 mutation for the customer

### CODE REVIEW

File: es6-lib/config/config.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ default {
   //gate at which the service will not read the whole file to generate
   //a detailed summary. for files over this limit, an abbreviated summary
   //will be returned, which is much faster, but less helpful
-  abbreviateSummarySize: 500 * 1000, //500kb
+  abbreviateSummarySize: 1500 * 1000000, //1.5gb
```
Comment (kaidamasaki) [inline es6-lib/config/config.js]: I worry a little about this number only being 1.5 times the nginx limit we have right now.

I might bump this even higher.

**Comment by rantler** [comment]
This might interact with the `NGINX_CLIENT_MAX_BODY_SIZE` in the `platform-ui/frontend/config/nginx.conf.j2` file. It is currently set to `8m` in `apps-marathon/resources/frontend.json`.

**Comment by kaidamasaki** [comment]
@rantler: The point was to make sure this was higher than the nginx 1gb upload.

The 8mb limit won't apply here anyway, just the 1gb limit since it's an upload.

**Comment by catstavi** [comment]
Has a ticket now: EN-17300

**Comment by rantler** [comment]
The sidecar nginx process _inside_ the frontend containers _does_ limit the upload body size. This might actually apply here. It'll work when testing locally unless you run the frontend docker container on your dev box.

**Comment by kaidamasaki** [comment]
If the limit applies to uploads then geo-import would be broken in production.  This doesn't change the size of allowed files, just the size at which we switch into "summarization mode".

**Pull request merged**